wordpress-mobile/WordPress-iOS
# Feature/about dismiss button

Implements #17538. This PR adds the close button to the about screen.

<img src="https://user-images.githubusercontent.com/4780/143233615-cbff6ffc-c03f-4fdb-9b8d-55da1dc14b2e.png" width="350">

I didn't make the metrics for the close button customizable at this stage as it'd make code for the host app a little more complex and at the moment we don't know that any app would want to customize those things. I figured it would be better to make it customizable in the future should the need arise.

**To test**

* Build and run
* Navigate to About, check you can see the about button
* Tap the About button and check it dismisses the screen

## Regression Notes
1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
